### PR TITLE
Add github actions on push/pr

### PR DESCRIPTION
**Pull request by recycletechno**
Some tests do not run (test_swap_single, test_swap_multi)

### CODE REVIEW

File: .github/workflows/python-app.yml [hunk 1]
```diff
@@ -46,34 +43,55 @@ jobs:
...
-        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+        flake8 --exclude=SimSwap . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+    - name: Check status
+      run: |
+        echo "PYTHONPATH: $PYTHONPATH"
+        echo "Current directory: $(pwd)"
     - name: Test with pytest
       run: |
-        pytest
+        pytest --ignore=SimSwap
```
Comment (gurudevapp) [inline .github/workflows/python-app.yml:92]: please replace it with:
pytest --ignore=SimSwap -k "test_upload_image or test_upload_video or test_upload_multispecific or test_main_page"

This forces pytest execute only certain tests specified with -k argument

File: .github/workflows/python-app.yml [hunk 2]
```diff
@@ -46,34 +43,55 @@ jobs:
...
-        pip install insightface==0.2.1
-        pip install onnxruntime
-        pip install moviepy
-        pip install numpy~=1.21.6
         
-        # if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        pip install --upgrade pip
+        pip install -r requirements/git_actions_ubuntu.txt
+        
+        #pip install --upgrade pip
```
Comment (Illania) [inline .github/workflows/python-app.yml:64]: I guess this commented code duplicates requirements/git_actions_ubuntu.txt and can be deleted

**Review by gurudevapp — APPROVED** [review]
All looks perfect and works like a charm now! Great work, thank you!

**Review by Illania — APPROVED** [review]
I see all problems resolved now! Good job! Approved!

**Pull request merged**